portfolio/src/main/webapp/script.js

```diff
@@ -30,5 +30,6 @@ function addRandomFact() {
 async function getMessageFromServer() {
   const response = await fetch('/data');
   const quote = await response.text();
+  console.log(quote);
```

Comment:
console.log is good for testing, but shouldn't be included in submitted code :)

Reply:
Ok thanks that's good to know.